### DISCUSSION TITLE
Roadmap for 9.3.0 Release

### DISCUSSION THREAD

**Discussion by boast** [intlify/vue-i18n#1445] [General]
Hello all

It looks like the beta for 9.3.0 is now nearly a year old and I wanted to ask (because I could not find any information on this), what the current plan is on releasing a stable 9.3.0 version of this awesome vue plugin.

Currently we (= company and in private projects) rely on referencing the beta version in all our builds, especially because of the TS fix in beta.14 and the Vue 3.3 support in beta.18.

However, depending on beta versioned packages is problematic - you will not get the latest update if you do not specifically reference the beta version (i.e. `^9.0.0` will not work). It also requires for local renovate-bots to have access to GitHub to fully resolve the dependency tree, which can be problematic to internal repos which just cache npm. It's also a pain to manually update to the latest beta version, as we need to pin the version to prevent build errors on recommended packages.

I would gladly try help on blocking issues for the release, just point me to those issues. Or would it be possible to just increase the minor version just more frequently?

-- Dave

**Comment by luisdeka** [comment]
Same here, waiting for 9.3 release to, i have started using beta weeks ago and would love to stop it and use a fixed version

**Comment by volarname** [comment]
Hi
I have also checked https://vue-i18n.intlify.dev/guide/roadmap.html
but there are a lot of tasks and it's not clear what is part of 9.3.0 release
When will 9.3.0 land? What needs to be done?
I am interested specially for TS 5 support

**Comment by paescuj** [comment]
[9.3.0](https://github.com/intlify/vue-i18n-next/releases/tag/v9.3.0) is here 🥳
